src/makeCalculator.js

```diff
@@ -36,8 +36,41 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculatorObj = {
```

Comment:
```suggestion
  const calculator = {
```

Reply:
Ihor is right. Let's not use `obj` in variables names